Procedure/unit 1  kenney

mkenney2025 opened the pull request:
**Completed Work**
Task 1 - login information for the landing - this task set is ongoing, as I am receiving this information this week and will have time to get the right images to accompany it as well

Task 2 - Setting up and Breaking Down the Space for a Presentation Event - This task's text is mostly completed, and I am finalizing the graphics to accompany this as well. I just don't really know how well my file management is aligning right now, and didn't want to accidentally break anything by adding the images in quite yet

Task 3 - connecting technology has headings for organizing, text is still being written, and hashing out the materials and prerequisites


## User Scenario

Dr. Erika Smith is a club event organizer who runs campus organizations for neurodivergent students
She needs to understand the basics of the space to run a wellness event without trouble while creating a sensory-safe event for club members, and wants to learn how the screen interface works to create a private, quiet space for a curated event. Their biggest priorities are logging into the interface, dimming the lights and windows to minimize harsh lighting and sound in the space so that they can be flexible with the computers, consoles, and board games working in the space, and knowing how to orient the area to accommodate those needs.

jmathh commented on morrisb/Morris-PowerPoint-Speaker-Coach.md:
```diff
@@ -0,0 +1,41 @@
+
+1. Log into Microsoft PowerPoint
+   
+2. Open a presentation slide show file that you want to practice with.
```
This might be a place to add another screenshot?

jmathh commented on morrisb/Morris-PowerPoint-Speaker-Coach.md:
```diff
@@ -0,0 +1,41 @@
+   
+2. Click the ESC button on your keyboard to exit Speaker Coach.
+   
+3.  Your Rehersal Report will be generated.
```
Does this need to be a numbered step? I think this could be a bullet point.

jmathh reviewed (commented):
## Answering your questions:
>Is the audience clear? This is important on the level this procedure is written.

As I understand it from your user scenario and the procedure, the users are IT professionals who may/may not have used Speaker Coach before, but they are not beginners to the software or Microsoft Suite. Your audience wants to know how to use Speaker Coach to improve their presentation skills, but they don't need to be completely introduced to the software on a basic level. Does this seem accurate?

> Are the Notes helpful?

Yes! See my alerting comments for more detail.


> Are more screenshots needed?

I suggest adding one more screenshot for step 2. Otherwise, I think you have enough and they highlight important info.


### Staging
The description of the software establishes the procedure and success condition for these instructions. The prerequisites are very important so it's great they're highlighted.

### Coaching
The steps are short and effective. It might be just me as someone unfamiliar with Speaker Coach, but steps 2-3 seem vague-- I think another screenshot could help clarify them?

### Alerting
The note at the end is very important and necessary. This prevents user confusion and frustration, particularly as your audience wants to study this report to improve their skills.

MadelynKenney committed:
Making the repository

This is to ensure my files are organized properly for the assignment. Content is being developed, but I don't want to mess up the workflow on anything.

MadelynKenney committed:
introduction, task 2 part 1 drafting

This includes the draft of a "welcome/introduction" page for the user guide, the outlining for task 1, and the first part of the task 2 text for setting up the landing for a presentation

MadelynKenney committed:
Renaming files/formatting/drafting

Renamed the files, added sentences to the introduction, added community guidelines, and deleted the copy template from my branch to get closer to aligning with the deliverables expectations for the end of the unit assignment.

MadelynKenney committed:
Renamed document

Renamed the document I think? I am losing track of what's happening, feeling a bit lost

MadelynKenney committed:
organization and readme

added to the readme document, and organized files in hopes of getting closer to assignment expectations

MadelynKenney committed:
Create PR-TEMPLATE.md
